app/models/spree_stripe/user_decorator.rb

```diff
@@ -0,0 +1,21 @@
+module SpreeStripe
+  module UserDecorator
+    def self.prepended(base)
+      base.after_update :update_stripe_customer, if: :should_update_stripe_customer?
+    end
+
+    def update_stripe_customer
+      SpreeStripe::UpdateCustomerJob.perform_later(id)
```

Comment:
It's not crucial when updating user/addresses so it's better to update it with a job